app/javascript/components/users/SignInComponent.spec.tsx

```diff
@@ -60,18 +62,51 @@ describe('SignInComponent', () => {
 		expect(userId).toBeEmptyDOMElement();
 		expect(error).toHaveTextContent('Not valid');
 	});
+});
 
-  // describe('SignInComponent', () => {
-  //   it('signIn successfully', async() => {
-  //   const initialValues = {
-  //       email: 'houdini@houdini.com',
-  //       password: 'mkamkamkamka',
-  //   };
-
-  //   expect(initialValues).toHaveTextContent("houdini@houdini.com");
-  // });
-  
+/** Test that SignInComponent has email and password input fields, and a submit button  */
+describe('SignInComponent', () => {
```

Comment:
Please don't use enzyme. You should test this using the React testing library https://testing-library.com/docs/react-testing-library/intro/

Reply:
Sounds good. I'll remove it